submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/integration/MongoDbReposConfig.kt

```diff
@@ -0,0 +1,33 @@
+package ac.uk.ebi.biostd.persistence.doc.integration
+
+import ac.uk.ebi.biostd.persistence.doc.MongoDbConfig
+import ac.uk.ebi.biostd.persistence.doc.db.data.SubmissionDocDataRepository
+import ac.uk.ebi.biostd.persistence.doc.db.data.SubmissionRequestDocDataRepository
+import ac.uk.ebi.biostd.persistence.doc.db.repositories.SubmissionMongoRepository
+import ac.uk.ebi.biostd.persistence.doc.db.repositories.SubmissionRequestRepository
+import ac.uk.ebi.biostd.persistence.doc.service.SubmissionMongoMetaQueryService
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+import org.springframework.data.mongodb.core.MongoTemplate
+
+@Configuration
+@Import(MongoDbConfig::class)
+@ConditionalOnProperty(prefix = "app.persistence", name = ["enabledMongo"])
```

Comment:
As I suggested in the previous PR. The property name should be either `enableMongo` or `mongoEnabled`.

Also, we need to update the file `application-local.yml` with the new properties.